cli/src/wallet_cli/mod.rs

```diff
@@ -325,6 +327,13 @@ pub enum WalletCommand {
     },
     /// List the unspent outputs.
     UnspentOutputs,
+    /// Get information of a validator.
+    Validator { account_id: AccountId },
+    /// List all validators known to the node.
+    Validators {
+        page_size: Option<u32>,
```

Comment:
Let's not have pagination stuff in the CLI?

Reply:
hmm, probably not :+1:

Reply:
removed